02_configure_host.sh

```diff
@@ -224,6 +224,12 @@ else
     # Create a blank authfile in order to have something valid when we read it in 04_setup_ironic.sh
     echo '{}' | sudo dd of=${REGISTRY_CREDS}
 fi
+# Since podman 2.2.1 the REGISTRY_CREDS file gets written out as
+# o600, where as in previous versions it was 644 - to enable reading
+# as $USER elsewhere we chown here, but in future we should probably
+# consider moving all podman calls to rootless mode (e.g remove sudo)
+sudo chown $USER:$USER ${REGISTRY_CREDS}
+ls -l ${REGISTRY_CREDS}
```

Comment:
Not hurting anything, but did you mean to leave this?

Reply:
I was thinking it could potentially be useful debug data in the event the ownership changes again on the next podman upgrade, but I guess that's true of a lot of things :smile: 

I can remove it if folks think it's just noise :+1:

Reply:
Nah, that's fine. It just looked like it might be a debug statement that was left accidentally. It's only one line of output. :-)